source/meshworker/scratch_data.cc

```diff
@@ -291,6 +291,18 @@ namespace MeshWorker
 
 
 
+  template <int dim, int spacedim>
+  const FEInterfaceValues<dim, spacedim> &
+  ScratchData<dim, spacedim>::get_current_interface_fe_values() const
+  {
+    Assert(interface_fe_values != nullptr,
+           ExcMessage("You have to initialize the cache using one of the "
```

Comment:
Maybe we could be specific about which reinit function to call? Only the one that takes 6 arguments (cell, face, subface, neighbor_cell, neighbor_face, neighbor_subface) will initialize an FEInterfaceValues object. I think this would make the errors a bit more understandable.

Reply:
No problem, but may we please do this as a follow up?